yarn.lock

```diff
@@ -6663,7 +6670,7 @@ cross-fetch@^2.1.0:
     node-fetch "2.6.1"
     whatwg-fetch "2.0.4"
 
-cross-spawn@7.0.3, cross-spawn@^7.0.0, cross-spawn@^7.0.2:
+cross-spawn@7.0.3, cross-spawn@^7.0.0, cross-spawn@^7.0.1, cross-spawn@^7.0.2:
```

Comment:
you removed the caret but need to regenerate the lockfile :)

Reply:
Since this is a subdependency from 'cross-env', it depends on whether its pinned on there repo. Is this correct @turtlemoji ?

Did `yarn install` again but yarn.lock remains as is.